pkgs/applications/audio/melmatcheq.lv2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkgconfig }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, xorg, xorgproto, cairo, lv2, pkg-config }:
```

Reply:
Ah, yes, thanks.
Out of curiosity: is there a reason to not do a tree-wide replace of ``pkgconfig`` for ``pkg-config``, except for the alias?
I usually start new pkgs by copying a similar existing one and adapting it, so this kind of error is easy to creep in.
I can't imagine I'm the only one doing this.